package.json

```diff
@@ -25,7 +25,7 @@
     "eslint-plugin-react": "^6.10.0",
     "eslint-plugin-require-path-exists": "^1.1.5",
     "husky": "^0.13.1",
-    "inferno": "^1.4.2",
+    "inferno": "^1.3.0-rc.3",
```

Comment:
was it intentional or accidental? why did you decide to change version back to 1.3?